core/src/main/kotlin/org/evomaster/core/search/service/FitnessFunction.kt

```diff
@@ -138,12 +138,14 @@ abstract class FitnessFunction<T>  where T : Individual {
         actionsSize: Int
     ) : EvaluatedIndividual<T>?{
 
+        val allTargetsWithDescriptive = config.processFormat == EMConfig.ProcessDataFormat.TARGET_HEURISTIC
```

Comment:
add some comments here, to explain "why" this is done. ie, by default we optimize for performance in collecting coverage values, but for special cases we want to collect full info